frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx

```diff
@@ -112,13 +112,15 @@ export const RecentEventsBodyContent: React.FC<RecentEventsBodyContentProps> = (
         ))}
       </Accordion>
       {sortedEvents.length > 50 && !!moreLink && (
-        <Link
-          className="co-activity-card__recent-more-link"
-          to={moreLink}
-          data-test="events-view-all-link"
-        >
-          {t('console-shared~View all events')}
-        </Link>
+        <CardFooter className="co-activity-card__recent-footer">
+          <Link
+            className="co-activity-card__recent-more-link"
+            to={moreLink}
+            data-test="events-view-all-link"
+          >
+            {t('console-shared~View all events')}
```

Comment:
the fact we truncate the events to the latest 50 isn't mentioned, is this worth mentioning?